### PR TITLE
Add migration guide for cosmos, storage and storage queue modules 

### PR DESCRIPTION
**Pull request by backwind1233**

### CODE REVIEW

File: sdk/spring/migration-guide.md [hunk 1]
```diff
@@ -147,6 +147,136 @@ For a full list of common configurations, check this list **[placeholder]**.
...
+
+
+### Migration steps of azure-spring-boot-starter-storage
+**Overview**
+The artifact `azure-spring-boot-starter-storage` was divided into two artifacts: `spring-cloud-azure-starter-storage-blob`,`spring-cloud-azure-starter-storage-file-share`.
+
+| Artifact Id                                   | Azure Service      |
+|-----------------------------------------------|--------------------|
+| **spring-cloud-azure-starter-storage-blob**       | [Azure Storage Blob](https://docs.microsoft.com/azure/storage/blobs/) |
+| **spring-cloud-azure-starter-storage-file-share** | [Azure Storage File](https://docs.microsoft.com/zh-cn/azure/storage/files/) |
```
Comment (stliu) [inline sdk/spring/migration-guide.md:192]: you should not give a *zh-cn* link

File: sdk/spring/migration-guide.md [hunk 2]
```diff
@@ -147,6 +147,136 @@ For a full list of common configurations, check this list **[placeholder]**.
 
 ### Each SDK configurations
 
+## Migration steps of each SDK
```
Comment (saragluna) [inline sdk/spring/migration-guide.md:150]: This whole section is for configuration changes, there's one section about artifact changes above.

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/spring/migration-guide.md (hunk 2 above)
Comment (backwind1233) [reply sdk/spring/migration-guide.md:150]: I think it shoud not under  `## Configuration properties` section, 
I don't know where to put it, so I create a new section `## Migration steps of each SDK`.

File: sdk/spring/migration-guide.md (hunk 1 above)
Comment (backwind1233) [reply sdk/spring/migration-guide.md:192]: yes，updated

**Comment by backwind1233** [comment]
close this pr, the content is migrate to this [wiki](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Cloud-Azure-Migration-Guide-for-4.0)

**Comment by backwind1233** [comment]
close this pr, the content is migrate to this [wiki](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Cloud-Azure-Migration-Guide-for-4.0)

**Pull request closed without merging**